lib/Elevate/Blockers/Databases.pm

```diff
@@ -185,6 +187,25 @@ sub _blocker_mysql_upgrade_in_progress ($self) {
     return 0;
 }
 
+sub _blocker_mysql_governor ($self) {
+
+    if ( Cpanel::Pkgr::is_installed('governor-mysql') ) {
+        return $self->has_blocker( <<~'EOS' );
+You have MySQL Governor installed.  Upgrades with this software in place are not supported.
+For more information regarding MySQL Governor, please review the documentation:
+
+    https://docs.cloudlinux.com/shared/cloudlinux_os_components/#mysql-governor
+
+To remove MySQL Governor, execute the following command:
```

Comment:
I don't think we should recommend removal. What about. "We will be adding support for governor soon. Stay tuned!"

Reply:
I would rather not put anything about adding support soon in case Anne finds blockers while testing it that requires us to leave the blocker in place until CL can address an issue on their end.

I could just leave this part off entirely and have it end with the 'for more information' line.  Would you be okay with that?

Reply:
yep. just block with no advice.